webapp/channels/src/components/dot_menu/dot_menu.tsx

```diff
@@ -415,8 +415,9 @@ export class DotMenuClass extends React.PureComponent<Props, State> {
         const isFollowingThread = this.props.isFollowingThread ?? this.props.isMentionedInRootPost;
         const isMobile = this.props.isMobileView;
         const isSystemMessage = PostUtils.isSystemMessage(this.props.post);
+        const isBurnOnReadPost = this.props.post.type === PostTypes.BURN_ON_READ;
```

Comment:
We have two ways of checking this - direct type comparison like this, and we also have the `isBurnOnReadPost` selector. We should stick with just one way and either is fine.